pom.xml

```diff
@@ -31,7 +31,7 @@
 		<frontend.version>v6.0.25</frontend.version>
 
         <!-- THIS SHOULD BE KEPT IN SYNC TO VERSION IN CGDS.SQL -->
-		<db.version>2.13.1</db.version>
+		<db.version>2.14.0</db.version>
```

Comment:
I had the impression that @dippindots believed that because we were doing something which was backwards compatible (only expanding the range of an integer field) that this change could be a simple point release increment rather than a minor number increment. But whatever makes sense ... I think this case is kind of "in between" the two examples given in the guidelines. We are not renaming a table or a field ... but we are altering a field (which would require an installer to update their deployed cBioPortal backend at the time of migration). Under that consideration maybe it is a minor number increment.